### PR TITLE
enh: add support for dask

### LINKED ISSUE #132: Plugin - DASK distributed
return dask support to pydra

### PR DESCRIPTION
**Pull request by satra**
## Types of changes
- [X] New feature (non-breaking change which adds functionality)

## Summary
- adds support for dask
- closes #132 

## Checklist

- [ ] All tests passing
- [X] I have added tests to cover my changes
- [ ] I have updated documentation (if necessary)
- [X] My code follows the code style of this project

## Acknowledgment
- [X] I acknowledge that this contribution will be available under the Apache 2 license.

### CODE REVIEW

**Comment by satra** [comment]
@djarecka - this seems to run on my machine in general. had to set `ulimit -n 65535` on my terminal to avoid too many open files error.

**Comment by djarecka** [comment]
@satra - i will check it again tomorrow, but at some point increasing ulimit stopped working for me

File: setup.cfg [hunk 1]
```diff
@@ -66,7 +66,7 @@ test =
     pytest-cov
     pytest-env
     pytest-xdist
-    distributed
+    dask
```
Comment (djarecka) [inline setup.cfg:69]: i believe distributed is needed anyway

**Comment by satra** [comment]
i had to remove the restriction on tornado. so essentially the setting that works with jupyter does not work with dask.  also dask seems finnicky in terms of the number of tests that are run, and loses communication with scheduler. overall, i think this works, but we need to perhaps mark it as an experimental feature for now.

**Comment by djarecka** [comment]
yes, I'm able to run single tests, but not able to run all...:( i'm also having issue with setting limit to as high as 65535.

**Comment by djarecka** [comment]
@satra - let me know if you want me to prepare a set of tests for dasks (mostly to not run everything at once) and merge it

**Comment by satra** [comment]
@djarecka - sure. i know it works, but i think we should still label as experimental support at this point.

**Pull request merged**